app/scripts/views/force_auth.js

```diff
@@ -89,8 +115,12 @@ define(function (require, exports, module) {
       var email = self.relier.get('email');
 
       return self.resetPassword(email)
-        .fail(function (err) {
-          self.displayError(err);
+        .fail(function (error) {
+          if (AuthErrors.is(error, 'UNKNOWN_ACCOUNT')) {
```

Comment:
@shane-tomlinson I believe we are already handling that well. I think it makes sense to keep the user on the reset page (in case they mistyped their email).
<img width="494" alt="screenshot 2016-02-02 10 43 20" src="https://cloud.githubusercontent.com/assets/68213/12754616/d94aeca4-c999-11e5-92e7-d5e780f77492.png">